### ISSUE TITLE
[Question] How to call functions mentioned in dynamic components

### ISSUE THREAD

**Issue by sanket-thotange** [gund/ng-dynamic-component#487]
I am trying to call function named 'autoRefreshData' traversing through all the components.
So I have an array of components which are stored in components variable.

`
this.components.forEach((c)=> c.inputs.until = new Date())
`
This way I was able to change the inputs, but how to call method/functions from dynamic components?

**Comment by sanket-thotange** [comment]
I figured it out

`@ViewChildren('dynamicComponents') dc: any;`

`dc._results.forEach((c)=> c.componentRef.instance.autoRefreshData());`

**Comment by gund** [comment]
Yes that's one way to do it, and also if you are rendering components via `ndc-dynamic` component then you can use it's output 
`ndcDynamicCreated` to get `ComponentRef` instance and call it's method.
